### PR TITLE
Update the release notes to give credits to our contributors

### PR DESCRIPTION
**Pull request by abstractj**
- [KEYCLOAK-11193] Add new  option to JS adapter
- [KEYCLOAK-9786] Secure token and logout endpoint

### CODE REVIEW

File: release_notes/topics/8_0_0.adoc [hunk 1]
```diff
@@ -25,3 +25,9 @@ file was also provided.
 Notice that when {project_name} acts as the IDP we can use the built-in role mappers to perform any necessary mappings
 before setting the roles into the assertion, so this SPI will probably be redundant in this case. The `RoleMappingsProvider`
 SPI was designed for situations when the IDP offer no way to map roles before adding them to the assertion.
+
+= Other improvements
+
+* Added option to disable logging to the console. Thanks to https://github.com/jonkoops[jonkoops]
```
Comment (mhajas) [inline release_notes/topics/8_0_0.adoc]:
```suggestion
* Added option to enable/disable logging to the console. Thanks to https://github.com/jonkoops[jonkoops]
```

**Review by mhajas — CHANGES_REQUESTED** [review]
Just one comment, the change was actually to disable logging and enable it only in case a parameter is set to true

**Review by andymunro — DISMISSED** [review]

**Review by mhajas — DISMISSED** [review]

**Review by stianst — APPROVED** [review]

File: release_notes/topics/8_0_0.adoc [hunk 2]
```diff
@@ -25,3 +25,10 @@ file was also provided.
 Notice that when {project_name} acts as the IDP we can use the built-in role mappers to perform any necessary mappings
 before setting the roles into the assertion, so this SPI will probably be redundant in this case. The `RoleMappingsProvider`
 SPI was designed for situations when the IDP offer no way to map roles before adding them to the assertion.
+
+= Other improvements
+
+* Added option to enable/disable logging to the console. Thanks to https://github.com/jonkoops[jonkoops]
```
Comment (stianst) [inline release_notes/topics/8_0_0.adoc]: This should mention that it is for the JS adapter

**Review by stianst — CHANGES_REQUESTED** [review]

**Comment by abstractj** [comment]
@stianst updated.

**Review by andymunro — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**